templates/master/00-master/_base/files/usr-local-share-openshift-recovery-template-kube-etcd-cert-signer-yaml-template.yaml

```diff
@@ -0,0 +1,77 @@
+filesystem: "root"
+mode: 644
```

Comment:
`0644` (that's likely why ci is failing) use octal in the other files as well

Reply:
nice catch updated.